definitions.yaml

```diff
@@ -596,6 +596,33 @@ definitions:
               awsoperator:
                 type: string
                 description: ARN of the IAM role to assume by the software operating clusters
+      azure:
+        type: object
+        description: Credentials specific to an Azure service principal
+        required:
+          - credential
+        properties:
```

Comment:
- `clientID` should become `client_id`
- `secredID` should become `secret_id`
- `subscriptionID` -> `subscription_id`
- `tenantID` -> `tenant_id`

Reply:
fixed